gladier_xpcs/flow_gpu.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import time
+import random
+import globus_sdk
+from gladier import GladierBaseClient, generate_flow_definition, utils
+# import gladier_xpcs.log  # Uncomment for debug logging
+
+
+log = logging.getLogger(__name__)
+
+@generate_flow_definition(modifiers={
+   'publish_gather_metadata': {'payload': '$.GatherXpcsMetadata.details.result[0]'}
+})
+class XPCSGPUFlow(GladierBaseClient):
+    globus_group = '368beb47-c9c5-11e9-b455-0efb3ba9a670'
+    max_retries = 10
+    containers = {}
+
+    gladier_tools = [
+        'gladier_xpcs.tools.TransferFromClutchToTheta',
+        'gladier_xpcs.tools.PrePublish',
+        'gladier_xpcs.tools.AcquireNodes',
+        'gladier_xpcs.tools.EigenCorrGPU',
+        'gladier_xpcs.tools.MakeCorrPlots',
+        'gladier_xpcs.tools.gather_xpcs_metadata.GatherXPCSMetadata',
+        'gladier_xpcs.tools.Publish',
+    ]
+
+    def register_funcx_function(self, function):
```

Comment:
This needs to go to base soon.

Reply:
Going to go ahead and resolve this, from the comment below